.github/workflows/build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: |
           git clone https://github.com/rust-bitcoin/rust-lightning
           cd rust-lightning
-          git checkout 0.0.110-bindings
+          git checkout 0.0.111-bindings
```

Comment:
why do you do the checkout as a separate command instead of just `git clone --branch`?